db/migrate/20180410125416_rename_ansible_job_categories.rb

```diff
@@ -1,5 +1,9 @@
 class RenameAnsibleJobCategories < ActiveRecord::Migration[5.1]
   def up
+    unless User.unscoped.find_by_login(User::ANONYMOUS_ADMIN)
+      puts "No ANONYMOUS_ADMIN found. Skipping renaming Ansbible jobs"
```

Comment:
I'll fix this on merge 'Ansible' - :smile:

Reply:
Thanks! One of the more creative typos. :D